src/server/rollbar.js

```diff
@@ -1,7 +1,8 @@
 var util = require('util');
 var os = require('os');
+var path = require('path');
 
-var packageJson = require('../../package.json');
+var packageJson = require(path.join(__dirname, '../../package.json'));
```

Comment:
Why does the path only resolve incorrectly here, and not for any of the following relative paths?

Reply:
I think because it's the `package.json` file and it's not included in the dist.

Reply:
However, using `path.join` for dynamic imports won't work with `esbuild`, as it doesn't support dynamic imports yet (https://github.com/evanw/esbuild/issues/700).